### DISCUSSION TITLE
Tests for Editable Property

### DISCUSSION THREAD

**Discussion by eliotwrobson** [PrairieLearn/PrairieLearn#6040] [Q&A] [answered]
In looking at the discussion in #5540, I realized that the custom elements for our course don't have support set up for the editable property. Is there an existing assignment configuration that makes it easy to test for this? I don't think we've had any issues about this come up yet, but we'd like to resolve them preemptively.

**Comment by jonatanschroeder** [comment, marked as answer]
An element will typically be disabled if the student is not expected to change it anymore. This can happen in exams if the exam is closed, or in questions with limited attempts per variant if the number of attempts has been exhausted, or if the assessment's current access rule has `active` set to false. So you should be able to test this feature by viewing as student, then opening an exam-type assessment, trying some answers and closing the exam (Finish Assessment). After this point, if you open the instance questions, most elements will be disabled.
